lib/topological_inventory/orchestrator/targeted_update.rb

```diff
@@ -72,6 +75,14 @@ def sync_targets_with_openshift
 
       private
 
+      def enable_supported_source_types
+        @targets.each do |target|
+          if (source_type = target[:source_type])
```

Comment:
Does this need to be an `==`

Reply:
re-worked it to remove the inline assignment.